dwave/system/composites/embedding.py

```diff
@@ -31,171 +33,140 @@
-                reports the fraction of chains broken before unembedding.
+                If True, the unembedded response contains a
+                ‘chain_break_fraction’ field that reports the fraction of chains
+                broken before unembedding.
+
+            embedding_parameters (dict, optional):
+                If provided, parameter are passed to the embedding method as
+                keyword arguments. Overrides any `embedding_parameters` passed
+                to the constructor.
 
             **parameters:
-                Parameters for the sampling method, specified by the child sampler.
+                Parameters for the sampling method, specified by the child
+                sampler.
 
         Returns:
-            :class:`dimod.SampleSet`: A `dimod` :obj:`~dimod.SampleSet` object.
-
-        Examples:
-            This example submits an triangle-structured Ising problem to a D-Wave solver, selected
-            by the user's default
-            :std:doc:`D-Wave Cloud Client configuration file <cloud-client:intro>`,
-            by minor-embedding the problem's variables to physical qubits.
-
-            >>> from dwave.system.samplers import DWaveSampler
-            >>> from dwave.system.composites import EmbeddingComposite
-            >>> import dimod
-            ...
-            >>> sampler = EmbeddingComposite(DWaveSampler())
-            >>> bqm = dimod.BinaryQuadraticModel.from_ising({}, {'ab': 0.5, 'bc': 0.5, 'ca': 0.5})
-            >>> response = sampler.sample(bqm, chain_strength=2)
-            >>> response.first:    # doctest: +SKIP
-            Sample(sample={'a': -1, 'b': 1, 'c': 1}, energy=-0.5,
-                   num_occurrences=1, chain_break_fraction=0.0)
-
-        See `Ocean Glossary <https://docs.ocean.dwavesys.com/en/latest/glossary.html>`_
-        for explanations of technical terms in descriptions of Ocean tools.
+            :obj:`dimod.SampleSet`
+
         """
 
         # solve the problem on the child system
         child = self.child
 
         # apply the embedding to the given problem to map it to the child sampler
-        __, target_edgelist, target_adjacency = child.structure
+        __, target_edgelist, target_adjacency = self.target_structure
 
         # add self-loops to edgelist to handle singleton variables
         source_edgelist = list(bqm.quadratic) + [(v, v) for v in bqm.linear]
 
         # get the embedding
-        embedding = minorminer.find_embedding(source_edgelist, target_edgelist)
+        if embedding_parameters is None:
+            embedding_parameters = self.embedding_parameters
+        else:
+            # update the base parameters with the new ones provided
+            embedding_parameters.update((key, val)
+                                        for key, val in self.embedding_parameters
+                                        if key not in embedding_parameters)
```

Comment:
Because we don't want the change to affect the defaults provided at construction. Only to affect the embedding parameters in the `sample` method scope

Reply:
I will change the comment to make that clearer